apps/vs-code-designer/src/app/commands/GenerateStaticWebApp/generateStaticWebApp.ts

```diff
@@ -0,0 +1,11 @@
+import type { IAzureConnectorsContext } from '../workflows/azureConnectorWizard';
+import type * as vscode from 'vscode';
+import { executeCommand } from '../../utils/funcCoreTools/cpUtils';
+import { ext } from '../../../extensionVariables';
+
+export async function generateStaticWebApp(context: IAzureConnectorsContext, node: vscode.Uri): Promise<void> {
+  console.log('generate SWA');
```

Comment:
Since this is user facing can we make this more descriptive?

Reply:
@alain-zhiyanov Overall I would say to remove all console logs. If you want actually display something for the user to be aware of it, please the output channel and please localize the text. For example: 
` ext.outputChannel.appendLog(localize('excludedSettings', 'Excluded the following settings:'));`